modules/apps/frontend-data-set/frontend-data-set-views-web/src/test/utils/sortItems.spec.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import sortItems from '../../main/resources/META-INF/resources/js/utils/sortItems';
+import {IOrderable} from '../../main/resources/META-INF/resources/js/utils/types';
+
+/* items to order.
+	- id: allows itemOrder to refer to items in this array
+	- dateCreated: additional data that can be used to sort items
+*/
+const items = [
+	{
+		dateCreated: '2024-06-05T10:49:04Z',
+		id: 4,
+	},
+	{
+		dateCreated: '2024-06-05T10:49:03Z',
+		id: '3',
+	},
+	{
+		dateCreated: '2024-06-05T10:49:02Z',
+		id: 2,
+	},
+	{
+		dateCreated: '2024-06-05T10:49:01Z',
+		id: '1',
```

Comment:
@dsanz Not a blocker for this PR, but isn't storing IDs in order CSVs wrong? I thought we started moving them to ERCs while back :thinking: